### PR TITLE
Change repair appointment

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

File: compoments/report-repair/repair-appointment-changed-confirmation.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import ComponentHeader from '../componentHeader';
 import Link from 'next/link';
 
-const RepairAppointmentChangedConfirmation = ({ confirmationContact, newAppointmentSchedule }) => {
+const RepairAppointmentChangedConfirmation = ({ confirmation}) => {
```
Comment (Bhavesh-Patel) [inline compoments/report-repair/repair-appointment-changed-confirmation.js:6]: Nitpick: add space
```suggestion
const RepairAppointmentChangedConfirmation = ({ confirmation }) => {
```

File: compoments/report-repair/repair-availability.js [hunk 2]
```diff
@@ -12,37 +12,29 @@ import { serviceName } from '../../helpers/constants';
 import ErrorSummary from '../errorSummary';
 import {customerServicesTelephoneNumber} from '../../globals'
 import ComponentHeader from '../componentHeader';
-import { getRepairType } from '../../helpers/repairType';
 
-const RepairAvailability = ({ handleChange, values, fromDate }) => {
+const RepairAvailability = ({ handleChange, values ,fromDate, stepName, submitChangeAppointment, isSubmit=false}) => {
```
Comment (Bhavesh-Patel) [inline compoments/report-repair/repair-availability.js:16]: Nitpick: move comma
```suggestion
const RepairAvailability = ({ handleChange, values, fromDate, stepName, submitChangeAppointment, isSubmit=false}) => {
```

Comment (Bhavesh-Patel) [inline compoments/report-repair/repair-availability.js:16]: The prop `stepName` should be just name as this component doesn't understand the notion of a step.

File: pages/report-repair/[route].js [hunk 3]
```diff
@@ -104,7 +104,7 @@ function ReportRepair() {
     flow.nextStep(step, state, prevStep)
   }
 
-  const noBackLinkRequired = ['repair-cancelled-confirmation', 'repair-appointment-changed-confirmation']
+  const noBackLinkRequired = ['repair-cancelled-confirmation']
```
Comment (Bhavesh-Patel) [inline pages/report-repair/[route].js:107]: The previous version of this line ensures that the back link won't be shown for `repair-appointment-changed-confirmation'.
I believe this change needs reverting.

File: pages/report-repair/[route].js [hunk 4]
```diff
@@ -181,6 +181,7 @@ function ReportRepair() {
       if (response.ok) {
         setShowBack(false);
         router.push('repair-cancelled-confirmation');
+        setConfirmation(values.repairAppointmentDetails.contactDetails.value);
```
Comment (Bhavesh-Patel) [inline pages/report-repair/[route].js:184]: This is not required as the cancellation page (no longer) shows the contact details.

(I feel this and the above [comment](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/107/files#diff-54de87070147bd6483cb396dde9b615c82d0684d7ef8dd228e327ea5b0d36ddcR107) may be accidental and caused by merge conflicts)

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]

**Review by markWarr — COMMENTED** [review]
Cypress test is still failing: 

1) repair appointment changed
       Content
         does not display back link:
     AssertionError: Timed out retrying after 4000ms: Expected not to find content: 'Back' within the element: [ <a.govuk-skip-link.govuk-link--no-visited-state>, 6 more... ] but continuously found it.
      at Context.eval (http://localhost:3000/__cypress/tests?p=tests/cypress/integration/reportRepair/repairAppointmentChangedConfirmation.spec.js:125:36)

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**